lighthouse-core/audits/accessibility/axe-audit.js

```diff
@@ -39,11 +42,34 @@ class AxeAudit extends Audit {
       };
     }
 
-    const violations = artifacts.Accessibility.violations || [];
-    const rule = violations.find(result => result.id === this.meta.id);
+    // Detect errors reported within aXe 'incomplete' results
+    // aXe uses this result type to indicate errors, or rules which require manual investigation
+    // If aXe reports an error, then bubble it up to the caller
+    const incomplete = artifacts.Accessibility.incomplete || [];
+    const isIncomplete = incomplete.find(result => result.id === this.meta.id);
+    if (isIncomplete && isIncomplete.error) {
+      return Audit.generateErrorAuditResult(this, isIncomplete.error.message);
+    }
+
+    const isInformative = this.meta.scoreDisplayMode === Audit.SCORING_MODES.INFORMATIVE;
+    const failureCases = [
+      ...(artifacts.Accessibility.violations || []),
+      ...(artifacts.Accessibility.incomplete || []).filter(() => isInformative),
```

Comment:
I'm not sure we need this filter now, maybe we can get away with something more like...


```js
const violations = artifacts.Accessibility.violations || [];
const failureCases = isInformative ? violations.concat(informative) : violations;